app/build.gradle

```diff
@@ -124,7 +124,7 @@ dependencies {
       'androidx.work:work-runtime-ktx:2.4.0',
       'com.github.bumptech.glide:glide:4.11.0',
       'com.google.android.flexbox:flexbox:3.0.0',
-      'com.google.android.material:material:1.2.0-alpha02',
+      'com.google.android.material:material:1.2.0',
```

Comment:
You also need to update the Bazel version of the app (i.e. by changing versions.bzl, and making sure that it builds locally--you'll likely need to repin dependencies for which there are instructions that will be outputted when you try to build after updating the version).

Reply:
I updated the bazel version but some tests are still failing

Reply:
It looks like everything's passing now in #4155.